setup.py

```diff
@@ -28,14 +28,10 @@
         "numpy",
         "rich",
         "rtree",  # Used by sumolib
-        "filelock",
-        "lz4",
-        "networkx",
-        "opencv-python",
```

Comment:
I think we can fully get rid of opencv but it is used in `benchmark` and `ultra`. We can either move it to a `[benchmark]` or `[rendering]` extra or give `benchmark` a setup file of its own. 

I also think it would be great if we could move panda3d out into an extra but that is enough work to make it a separate PR. @RutvikGupta could you make an issue for that?

Reply:
Ok sure.